package.json

```diff
@@ -28,7 +28,7 @@
     "reset": "pnpm clean && pnpm reset:modules",
     "reset:modules": "pnpm -r --parallel --include-workspace-root exec pnpm dlx rimraf node_modules pnpm-lock.yaml",
 
-    "storybook": "storybook dev -p 6010",
+    "storybook": "storybook dev -p 6006",
```

Comment:
Why the change?

Reply:
During Design System development, we'll be using storybooks for components. The one used for documentation development is secondary, which is why I've inverted them.